butter_cms/page.py

```diff
@@ -14,3 +14,13 @@ def all(self, page_type, params=None):
     def get(self, page_type, page_slug, params=None):
         full_slug = '{}/{}/'.format(page_type, page_slug)
         return self.api_get(slug=full_slug, params=params)
+
+    def search(self, query, params=None):
```

Comment:
Why not set default value for `params` to be `{}`?

Reply:
The default values for function arguments should never be mutable (like `[]` or `{}`), as you can get unexpected behaviour that way.